src/features/persistence/github-gist.ts

```
@@ -0,0 +1,72 @@
+interface GistFile {
+    content: string
+}
+
+interface Gist {
+    description: string
+    public: boolean
+    files: Record<string, GistFile>
+}
+
+export const githubGistService = {
+    async saveConfig(config: any, token: string) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)